android/KMEA/app/src/main/java/com/tavultesoft/kmea/KeyboardInfoActivity.java

```diff
@@ -150,6 +156,23 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
         }
       }
     });
+
+    // If QRGen library included, also display QR code for sharing keyboard
+    if (QRCodeUtil.exists(context)) {
+      String url = String.format("%s%s%s", QRCodeUtil.QR_BASE, kbID, "/share");
```

Comment:
Could we change this around to something like the following:
```suggestion
      String url = String.format(QRCodeUtil.QR_SHAREURL, kbID);
```

Where `QRCodeUtil.QR_SHAREURL="https://keyman.com/go/keyboard/%s/share"`. That results in fewer embedded strings to maintain.